INSTALL.md

```diff
@@ -4,6 +4,7 @@
 * [Get the source code](#get-the-source-code)
 * [Dependencies](#dependencies)
   * [Linux dependencies](#linux-dependencies)
+* [Building Skia dependency](#building-skia-dependency-os-xlinuxbsd)
```

Comment:
Why not? Its a source of confusion when building the project. The library dependency, unlike allegro, isn't available through package managers. It also requires a third party tool (depot tools).

In lieu of that, is linking to the installation instructions on the Skia website OK?